packages/arb-token-bridge-ui/src/components/TransactionHistory/helpers.ts

```diff
@@ -45,7 +45,7 @@ export function isCctpTransfer(tx: Transfer): tx is MergedTransaction {
 
 export function isTxCompleted(tx: MergedTransaction) {
```

Comment:
maybe also explicitly state that the return type is boolean?

Reply:
done